docs/modules/ic-identity-guide/pages/auth-how-to.adoc

```diff
@@ -48,14 +50,14 @@ Authenticate using the method you selected when prompted.
 Your identity is not created until you perform this step. 
 +
 At this point, depending on the device you are using, you might be asked to either use your device authenitication method, or to use an identity saved to your security key. If you are registering for the first time, choose to use the device authentication. 
-. Save your *ID number*.
+. Save your *User Number*.
 +
-When you have registered your device, you’ll receive an *ID number*.
+When you have registered your device, you’ll receive a *User Number*. 
```

Comment:
See above.

Reply:
Occurrence references the text displayed in the UI. Has this changed yet?